FEATURES.md

```diff
@@ -146,3 +146,6 @@ entry in `config/features.json`.
      }
    });
   ```
+- `ds-deprecate-store-serialize` [#4654](https://github.com/emberjs/data/pull/4654)
+
+  Adds a deprecation warning when using Store#serialize method
```

Comment:
Let's also add a short sentence on what should be used instead. Just so people looking at the `FEATURES.md` immediately know what this is about.